src/Interfaces/ICircleBridgeProxy.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface ICircleBridgeProxy {
+    /// @notice Deposits and burns tokens from sender to be minted on destination domain.
+    /// @dev reverts if:
+    ///      - given burnToken is not supported.
+    ///      - given destinationDomain has no TokenMessenger registered.
+    ///      - transferFrom() reverts. For example, if sender's burnToken balance
+    ///        or approved allowance to this contract is less than `amount`.
+    ///      - burn() reverts. For example, if `amount` is 0.
+    ///      - MessageTransmitter returns false or reverts.
+    /// @param _amount Amount of tokens to burn.
+    /// @param _dstChid Destination domain.
+    /// @param _mintRecipient Address of mint recipient on destination domain.
+    /// @param _burnToken Address of contract to burn deposited tokens, on local domain.
+    /// @return nonce Unique nonce reserved by message.
```

Comment:
"..reserved by message" is not clear to me. What message?

Reply:
This is how the official CCTP contract describes that parameter: https://goerli.etherscan.io/address/0xd0c3da58f55358142b8d3e06c1c30c5c6114efe8#code
I would assume every transfer is packed as a message and this will be some identifier for it.